pubs/pubs_cmd.py

```diff
@@ -83,8 +83,15 @@ def execute(raw_args=sys.argv):
 
         # Eventually autocomplete
         autocomplete(parser)
+
         # Parse and run appropriate command
+        # if no command, print help and exit peacefully (as '--help' does)
         args = parser.parse_args(remaining_args)
+
+        if not args.command:
+            parser.print_help()
+            sys.exit(0)
```

Comment:
Here we would need to return an exit code `2` to be consistent with `argparse` and because this is still an error.

Reply:
`argparse._HelpAction.__call__` prints to stderr but then calls `argparse.ArgumentParser.exit()` which defaults to just a `sys.exit(0)`.  So when comparing exit codes of the `SystemExit`s that are raised, I thought this should be zero.  Is that wrong?

Reply:
What I meant is that not providing arguments should probably be considered an error (e.g. `git` returns with code `1` in the same situation). This is mainly to be consistent, so that a scripts _knows_ that the status is a failure.

On the other side we could consider that showing help is the default behavior and is not an error, but I think the other behavior is more consistent.

Reply:
Yep, you're right.  So do you think it should be error code `1` (like git) or `2`.  `1` seems more appropriate (operation not permitted).

Reply:
`2` is consistent with what we have for all the other argument errors.